bin/github-default-branch

```diff
@@ -12,6 +12,7 @@
     old: {type: "string", default: "master", description: "The name of the branch to rename"},
     new: {type: "string", default: "main", description: "The new branch name"},
     confirm: {type: "boolean", default: false, description: "Run without prompting for confirmation"},
+    keepDefault: {type: "boolean", default: false, description: "Retain the original default branch"}
```

Comment:
I think we talked about keeping the default whenever the default is not `master` automatically right @harrisonmuskat ?

So we don't need this